.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+name: Test
+
+on:
+  push
+
+jobs:
+
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
+        
+      - name: Prepare values
+        run: | 
+          echo GITHUB_ORGANISATION: fiware >> values.yaml
+          echo GITHUB_REPO: kong-plugins-fiware >> values.yaml
```

Comment:
I would add a comment that this is only 1.0.0 to fulfill the rockspec format, not the real 1.0.0